js/dashboard.js

```diff
@@ -209,7 +209,10 @@ var Dashboard = {
             var active = $(this).hasClass('active');
 
             if (active) {
-                const seconds = parseInt(CFG_GLPI.refresh_views || 30) * 60;
+                var seconds = parseInt(CFG_GLPI.refresh_views || 30) * 60;
+                if (seconds <= 0) {
+                    seconds = 60;
+                }
```

Comment:
If `CFG_GLPI.refresh_views` is configured to 0, it means that the `Never` value was defined in preferences.

In this case, using a high refresh rate (60s) does not seems logical to me. We should either disable autorefresh, either using the `30` minutes value that was expected to be set by the `|| 30` logic.
```suggestion
                var minutes = parseInt(CFG_GLPI.refresh_views);
                if (minutes == 0 || Number.isNaN(minutes)) {
                    minutes = 30;
                }
                var seconds = minutes * 60;
```

Reply:
> 

I'm on this issue